packages/swing-store/test/test-exportImport.js

```diff
@@ -281,6 +297,10 @@ async function testExportImport(
     ],
   ]);
 
+  expectedArtifactNames = Array.from(expectedArtifactNames);
+  expectedArtifactNames.push(`bundle.${bundleID1}`);
+  expectedArtifactNames.push(`bundle.${bundleID2}`);
```

Comment:
can we test that the new artifacts are readable, and that their contents match our expectations?

Reply:
The comparison of the before and after dumps should accomplish that.